modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -1952,10 +1954,18 @@ struct MapperUnit {
     I index0, index1;
 };
 
-inline static uint8_t calc(short alpha0, uint8_t src0, short alpha1, uint8_t src1) {
+CV_ALWAYS_INLINE uint8_t calc(short alpha0, uint8_t src0, short alpha1, uint8_t src1)
+{
     constexpr static const int half = 1 << 14;
     return (src0 * alpha0 + src1 * alpha1 + half) >> 15;
 }
+
+CV_ALWAYS_INLINE float calculation(float alpha0, float src0, float alpha1, float src1)
```

Comment:
IMHO: too common name

Reply:
Don't get why it's needed to be wrapped into separate function?

Reply:
@TolyaTalamanov It was wrapped into separate function for future. Case U8C1 will be implemented and [`calcRowLinear`](https://github.com/opencv/opencv/pull/21678/files#diff-7f0bda22bce310481b13ffb6c7114b9c957d864de2b95530dbe994b83c577d5bR2101) will become template for 2 types-`float` and `uchar` and so the `calculation` function will become one from two overloads.

Reply:
> IMHO: too common name

Changed.